rn-cli.config.js

```diff
@@ -14,6 +14,7 @@ const sharedBlacklist = [
   /node_modules[/\\]react[/\\]dist[/\\].*/,
   /website\/node_modules\/.*/,
   /heapCapture\/bundle\.js/,
+  /\.build[/\\]package.json/,
```

Comment:
Should we ignore the `.build` completely?

Reply:
No, we need to keep the package.json in git, so we can increment version number

Reply:
It's not `.gitignore`. I mean - should we ignore it completely in the `rn-cli`?

Reply:
Right, now I understood. Yes, we might as well do that.